Classes/Hooks/KeSearchIndexer.php

```diff
@@ -70,13 +70,35 @@ public function customIndexer(&$indexerConfig, &$indexerObject)
             $abstract = strip_tags($originalObject->getKeSearchAbstract($index));
             $content = strip_tags($originalObject->getKeSearchContent($index));
             $fullContent = $title . "\n" . $abstract . "\n" . $content;
+            $tags = [];
+            foreach ($originalObject->getCategories() as $category) {
```

Comment:
You are right `getKeSearchTags()` in the Interface is much cleaner (also add the implementation for the Calendarize Event model). So the developer get also the chance to add tags independent of the category relation....